src/pyOpenMS/tests/unittests/test_InstrumentSettings.py

```
@@ -0,0 +1,83 @@
+
+    def test_getAllNamesOfScanMode(self):
+        """Test getting all scan mode names."""
+        names = pyopenms.InstrumentSettings.getAllNamesOfScanMode()
+        self.assertGreater(len(names), 0)
+        # getAllNames returns bytes
+        self.assertIn(b"MS1Spectrum", names)
+        self.assertIn(b"MSnSpectrum", names)
+
+    def test_toScanMode_invalid(self):
+        """Test that invalid string raises exception."""
+        with self.assertRaises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)